MapboxNavigation/RouteVoiceController.swift

```diff
@@ -184,15 +159,16 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
         
         // If the user is merging onto a highway, an announcement to merge is a bit excessive
         if let upComingStep = routeProgress.currentLegProgress.upComingStep, routeProgress.currentLegProgress.currentStep.maneuverType == .takeOnRamp && upComingStep.maneuverType == .merge && routeProgress.currentLegProgress.alertUserLevel == .high {
-            return
+            return false
         }
         
-        if useDefaultVoice {
-            speakFallBack(fallbackText)
-        } else {
-            speakWithPolly(speechString(notification: notification, markUpWithSSML: true))
-        }
+        return true
+    }
+    
+    func process(_ notification: NSNotification) {
```

Comment:
This is a very vague name. Could we fold this into `alertLevelDidChange(_:)` and allow subclasses to override that method?